public/jsx/app.jsx

```diff
@@ -25,10 +27,23 @@ var App = React.createClass({
     var productId = router.getCurrentQuery().product;
 
     return {
+      error: AppStore.getError(),
       productId: productId,
     };
   },
 
+  componentDidMount: function() {
+    AppStore.addErrorListener(this.onError);
```

Comment:
You still need to wrap it (or a parent) with a connector though - same dependency. We'll still need a test that sets up the connector.